internal/server/v1beta1/asset.go

```diff
@@ -359,7 +361,16 @@ func (server *APIServer) upsertAsset(
 	mode string,
 	reqUpstreams,
 	reqDownstreams []*compassv1beta1.LineageNode,
-) (string, error) {
+) (id string, err error) {
+	defer func() {
+		server.assetUpdateCounter.Add(ctx, 1, metric.WithAttributes(
```

Comment:
just nice to have if we could test it and assert it properly using mock?

Reply:
Can't find any documentation or mock metric package for testing OpenTelemetry instrumentation. Will skip for now.